Makefile

```diff
@@ -115,10 +115,10 @@ grpc-plugin-storage-integration-test:
 .PHONY: test-compile-es-scripts
 test-compile-es-scripts:
 	docker run --rm -v ${PWD}:/tmp/jaeger python:3-alpine3.11 /usr/local/bin/python -m py_compile /tmp/jaeger/plugin/storage/es/esRollover.py
-	docker run --rm -v ${PWD}:/tmp/jaeger python:3-alpine3.11 /usr/local/bin/python -m py_compile /tmp/jaeger/plugin/storage/es/esCleaner.py
 
 .PHONY: index-cleaner-integration-test
-index-cleaner-integration-test: docker-images-elastic
+index-cleaner-integration-test:
+	$(MAKE) BASE_IMAGE=alpine:3.13 docker-images-elastic
```

Comment:
Curious why we are overwriting BASE_IMAGE here ? I would like to keep a single source of truth for all base images and debug images.

Reply:
I don't like that building the images requires running docker registry at localhost. This complicates local and CI setup.

Reply:
I think reverting this change back should fix the current CI failure.

 Here it is trying to tag the image name as alpine:3.13
```
make BASE_IMAGE=alpine:3.13 docker-images-elastic
make[1]: Entering directory '/home/runner/work/jaeger/jaeger'
docker buildx build -t alpine:3.13 --push \
	--build-arg root_image=alpine:3.13 \
	--build-arg cert_image=alpine:3.13 \
	--platform=linux/amd64,linux/s390x,linux/ppc64le,linux/arm64 \
	docker/base
```
which it later tries to push in the official alpine image resulting in
```
error: failed to solve: authorization status: 401: authorization failed
```

Reply:
yes this was the problem. I will try to play with tit and see if I could run it locally.

Can you run the full build locally? I had some issues with running the registry locally.

Reply:
@pavolloffay I had issues with local builds as well in the past; I managed to get it going I think. Maybe this helps? https://github.com/jaegertracing/jaeger/pull/2948/files#r666710826